src/inputs/dropdownMenu/dropdownMenuHeading.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import makeStyles from '../../styles/makeStyles';
+import Typography from '../../dataDisplay/typography';
+
+const propTypes = {
+    children: PropTypes.node.isRequired,
+    classes: PropTypes.shape({
+        root: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    classes: undefined,
+};
+
+const useStyles = makeStyles(({
+    spacing,
+}) => ({
+    root: {
+        padding: [[spacing(1), spacing(2), spacing(1)]],
+        '&:first-child': {
+            paddingTop: spacing(2),
+        },
+    },
+}));
+
+function DropdownMenuHeading(props) {
+    const {
+        children,
+    } = props;
+
+    const classes = useStyles(props);
+
+    return (
+        <Typography
+            classes={{
+                root: classes.root,
+            }}
+            color="textSecondary"
```

Comment:
Does this `color` prop need to be parameterized?  Like, might we want the ability to have different color specifications here for different usages?

**_Not a Blocker_** necessarily ... I'm just asking.  I trust your judgement.

Reply:
I say let's let it slide for now and we can update down the road if needed.

Reply:
Agreed!